courses/level101/big_data/evolution.md

```diff
@@ -45,41 +45,42 @@
 # Other tooling around Hadoop
 
 1. [**Hive**](https://hive.apache.org/)
-    1. Uses a language called HQL which is very SQL like.  Gives non-programmers the ability to query and analyze data in Hadoop.  Is basically an abstraction layer on top of map-reduce.
+    1. Uses a language called HQL which is very SQL like. Gives non-programmers the ability to query and analyze data in Hadoop. Is basically an abstraction layer on top of map-reduce.
     2. Ex. HQL query:
-        1. _SELECT pet.name, comment FROM pet JOIN event ON  (pet.name = event.name);_
+        1. `SELECT pet.name, comment FROM pet JOIN event ON (pet.name = event.name);`
     3. In mysql:
-        1. _SELECT pet.name, comment FROM pet, event WHERE  pet.name = event.name;_
+        1. `SELECT pet.name, comment FROM pet, event WHERE pet.name = event.name;`
 2. [**Pig**](https://pig.apache.org/)
-    1. Uses a scripting language called Pig Latin, which is more workflow driven.  Don't need to be an expert Java programmer but need a few coding skills.  Is also an abstraction layer on top of map-reduce.
+    1. Uses a scripting language called Pig Latin, which is more workflow driven. Don't need to be an expert Java programmer but need a few coding skills. Is also an abstraction layer on top of map-reduce.
     2. Here is a quick question for you:
-    What is the output of running the pig queries in the right column against the data present in the left column in the below image?
+    What is the output of running the Pig queries in the right column against the data present in the left column in the below image?
 
     ![Pig Example](images/pig_example.png)
 
     Output:
-    ```
+    <pre><code>
     7,Komal,Nayak,24,9848022334,trivendram
     8,Bharathi,Nambiayar,24,9848022333,Chennai
     5,Trupthi,Mohanthy,23,9848022336,Bhuwaneshwar
     6,Archana,Mishra,23,9848022335,Chennai
-    ```
+    </code></pre>
 
 3. [**Spark**](https://spark.apache.org/)
-    1. Spark provides primitives for in-memory cluster computing that allows user programs to load data into a cluster’s memory and query it repeatedly, making it well suited to machine learning algorithms.
+    1. Spark provides primitives for in-memory cluster computing that allows user programs to load data into a cluster’s memory and query it repeatedly, making it well-suited to machine learning algorithms.
 4. [**Presto**](https://prestodb.io/)
     1. Presto is a high performance, distributed SQL query engine for Big Data.
     2. Its architecture allows users to query a variety of data sources such as Hadoop, AWS S3, Alluxio, MySQL, Cassandra, Kafka, and MongoDB.
-    3. Example presto query:
-    ```
-    use studentDB;
-    show tables;
-    SELECT roll_no, name FROM studentDB.studentDetails where section=’A’ limit 5;
-    ```
+    3. Example Presto query:
+    <pre><code>
```

Comment:
have you verified this render as expected?

Reply:
Yes, I've verfied the rendered output.

[Live page](https://linkedin.github.io/school-of-sre/level101/big_data/evolution/)
 
![backquote](https://github.com/linkedin/school-of-sre/assets/94439978/04e388bf-66ac-42fe-97a5-5178f3a88ccd)

Locally rendered page

![pre code](https://github.com/linkedin/school-of-sre/assets/94439978/c3db468a-d7fb-45df-9bca-a60b85b9a8b1)